includes/definitions/quanta.yaml

```diff
@@ -6,9 +6,10 @@ over:
     - { graph: device_bits, text: 'Device Traffic' }
 discovery:
     - sysObjectId:
-        - .1.3.6.1.4.1.4413
-        - .1.3.6.1.4.1.7244
-    - sysDescr:
-        - vxworks
-        - quanta
-        - LB9, Runtime Code
+          - .1.3.6.1.4.1.7244
+    -
+      sysObjectId:
+          - .1.3.6.1.4.1.4413
+      sysDescr_regex:
+          - '/vxworks/i'
```

Comment:
We do have the one confirmed output from @tobiasa1 which validates VxWorks and the snmprec files we have look real.